### PR TITLE
Add target NBD_INFINITYAIO255

### PR DESCRIPTION
**Pull request by mywinchester**
Add target for `NEBD-NBD_INFINITYAIO255`, File already minimize and tested in hardware. 

Please notice me for any issues.

https://newbeedrone.com/collections/all/products/infinity-aio-255

### CODE REVIEW

**Comment by nerdCopter** [comment]
will need sync to https://github.com/betaflight/config as well.

**Comment by nerdCopter** [comment]
seems that GPS and LED_STRIP should not be enabled in the defines.

**Comment by mywinchester** [comment]
> seems that GPS and LED_STRIP should not be enabled in the defines.

Sorry for the latency, already fixed that in new commit.

**Comment by nerdCopter** [comment]
@haslinghuis , anything else this needs?

**Review by nerdCopter — DISMISSED** [review]

File: configs/default/NEBD-NBD_INFINITYAIO255.config [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+feature ESC_SENSOR
+
+# serial
+serial 1 131073 115200 57600 0 115200
+serial 2 2 115200 57600 0 115200
+serial 4 1024 115200 57600 0 115200
+serial 7 64 115200 57600 0 115200
+
+# master
+set mag_bustype = I2C
```
[the remark this answers is not in the document]
Comment (mywinchester) [reply configs/default/NEBD-NBD_INFINITYAIO255.config:92]: Thanks, I added it in latest commit, please check.

File: configs/default/NEBD-NBD_INFINITYAIO255.config [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+set blackbox_device = SPIFLASH
+set dshot_burst = ON
+set current_meter = ADC
+set battery_meter = ADC
+set ibata_scale = 100
+set beeper_inversion = ON
+set beeper_od = OFF
+set beeper_frequency = 5400
+set osd_canvas_width = 30
+set osd_canvas_height = 13
```
Comment (haslinghuis) [inline configs/default/NEBD-NBD_INFINITYAIO255.config:103]: canvas depends on build options.

```suggestion
```

Comment (haslinghuis) [inline configs/default/NEBD-NBD_INFINITYAIO255.config:103]: canvas depends on build options.

```suggestion
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**